### PR TITLE
Project routing 

### PR DESCRIPTION
**Pull request by PavelTitov16**
Added project routing via React Router

image

Also added simple routes for application

image
image

### CODE REVIEW

File: src/pages/AuthPage.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const AuthPage = () => {
+  return <h1>Auth Page</h1>
+}
```
Comment (KristiBo) [inline src/pages/AuthPage.tsx]: there's an empty line missing

File: src/pages/NotFoundPage.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const NotFoundPage = () => {
+	return <h1>404 - Page Not Found</h1>
+}
```
Comment (KristiBo) [inline src/pages/NotFoundPage.tsx]:
```suggestion
export const NotFoundPage = () => {
  return <h1>404 - Page Not Found</h1>
}

```

File: src/pages/HomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      </p>
+      <h1 className="text-4xl font-bold underline">
+        Hello Tailwind!
+      </h1>
+      <div>
+        <h2>{t('welcome')}</h2>
+      </div>
+    </>
+	)
+}
```
Comment (KristiBo) [inline src/pages/HomePage.tsx]: there's an empty line missing

File: src/pages/HomePage.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+
+import { RootState, increment, decrement } from '../store';
+
+import reactLogo from '../assets/react.svg';
+import viteLogo from '/vite.svg';
+
+export const HomePage = () => {
+	const count = useSelector((state: RootState) => state.counter.value);
```
Comment (KristiBo) [inline src/pages/HomePage.tsx]:
```suggestion
  const count = useSelector((state: RootState) => state.counter.value);
```

File: src/pages/HomePage.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        Click on the Vite and React logos to learn more
+      </p>
+      <h1 className="text-4xl font-bold underline">
+        Hello Tailwind!
+      </h1>
+      <div>
+        <h2>{t('welcome')}</h2>
+      </div>
+    </>
+	)
```
Comment (KristiBo) [inline src/pages/HomePage.tsx]: use 2 spaces

File: src/App.tsx [hunk 6]
```diff
@@ -1,40 +1,33 @@
...
+          </li>
+        </ul>
+      </nav>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/auth" element={<AuthPage />} />
+        <Route path="*" element={<NotFoundPage />} />
+      </Routes>
+    </Router>
+
```
Comment (KristiBo) [inline src/App.tsx]: unnecessary empty line

File: src/App.tsx [hunk 7]
```diff
@@ -1,40 +1,33 @@
...
-import viteLogo from '/vite.svg';
+import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom'
+
 import './App.css';
 
+import { HomePage } from './pages/HomePage';
+import { AuthPage } from './pages/AuthPage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
 function App() {
```
Comment (KristiBo) [inline src/App.tsx]: I prefer not to use default exports

Comment (PavelTitov16) [reply src/App.tsx]: completely agree

File: src/App.tsx (hunk 6 above)
Comment (PavelTitov16) [reply src/App.tsx]: thanks for notifying - fixed

File: src/pages/HomePage.tsx (hunk 5 above)
Comment (PavelTitov16) [reply src/pages/HomePage.tsx]: done :)

File: src/pages/HomePage.tsx (hunk 3 above)
Comment (PavelTitov16) [reply src/pages/HomePage.tsx]: fixed

File: src/pages/AuthPage.tsx (hunk 1 above)
Comment (PavelTitov16) [reply src/pages/AuthPage.tsx]: fixed

**Review by KristiBo — APPROVED** [review]
Good job, thanks 👍

**Review by turn2river — APPROVED** [review]

**Pull request merged**